### ISSUE TITLE
[Help]: Merging two different frames 

### ISSUE THREAD

**Issue by AmiraZakii** [gmtsar/gmtsar#862]
Hello,
Please I want to ask about the best way to merge the LOS velocity of three different sentinel-1 frames (they have overlapped areas) after generating by using GMTSAR even in radar or geographic coordinates.
Thanks

**Comment by Xiaohua-Eric-Xu** [comment]
You could use organize_files_tops[_linux].csh / create_frame_tops.csh / assemble_tops to create a new user-defined frame for TOPS data. 

If you mean some older satellites, e.g., ALOS, ERS, ENVISAT, then you need some dark magic (shell scripting) to do so.

**Comment by AmiraZakii** [comment]
@Xiaohua-Eric-Xu, thanks, but do you mean merge all sentinel-1 images before generating interferograms?! as I know organize_files_tops[_linux].csh organize the images?

**Comment by Xiaohua-Eric-Xu** [comment]
Yes, see section 3 in the attached document, which sort of describes how to reassemble the data. 
https://topex.ucsd.edu/gmtsar/tar/sentinel_time_series_5.pdf

**Comment by ditafaith** [comment]
Hi @Xiaohua-Eric-Xu, please allow me to open this issue.
Sometime we need to wait for a while to capture whole area of earthquake, since the earthquake is too large for single frame.
Then, how GMTSAR possible to merge final result (i.e., los_ll.grd) from 2 or more frames?
Is there any other procedure to merge them?

**Comment by Xiaohua-Eric-Xu** [comment]
If you mean merging along-track, you could use stitch_tops for processed SLC data or for Sentinel-1 TOPS data, you could use create_frame_tops.csh. If you mean merging for different tracks, we currently don't do so. The best way then I think is to create models or do decomposition when you have multiple observations from different look angles.

**Comment by ditafaith** [comment]
Hi @Xiaohua-Eric-Xu 
I meant, to merge from 2 Frames, with same orbit, e.g. Tibet earthquake 2025.

**Comment by Xiaohua-Eric-Xu** [comment]
You could run create_frame_tops.csh to get a few frames stitched together.
